src/net/JNetReflector/DotnetToolSettings.xml

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DotNetCliTool Version="1">
+	<Commands>
+		<Command Name="jnetreflector" EntryPoint="MASES.JNetReflector.dll" Runner="dotnet" />
```

Comment:
Maybe we have to simplify the command name. It is possible to add a second line with something like `jnr`?